src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -28,7 +29,7 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
-            base["SourceTimingDuration"] = string.Join(",", sourceTimings?.Select(e => e.TotalSeconds));
+            base["SourceTimingDuration"] = new JArray(sourceTimings.Select(e => e.TotalSeconds));
```

Comment:
I would recommend going right to string, unless the VS telem infra knows something special about `JArray`. `JsonConvert.SerializeObject(...)` would be simpler right?

Reply:
They pretty much do the same thing. 
They'd serialize slightly different as JArray.ToString() will put a newline after each object :) 

No need for the unneeded object though.

Reply:
Formatted Jason doesn't seem appropriate for this use case but it's up to you